### DISCUSSION TITLE
Lawnchair QuickSwitch Workaround for the ROMs That are Incompatible with QuickSwitch

### DISCUSSION THREAD

**Discussion by besomuncu** [LawnchairLauncher/lawnchair#4492] [Show and tell]
For LineageOS 21 - A14 (My ROM):

Step 1: Backup Trebuchet Launcher's APK

Step 2: Edit Lawnchair Launcher's package name to **"com.android.launcher3"** using an app you know (I used APK Editor)

*Deleted APK Editor right after I did this because the app is very sketchy, I recommend this to you too*

Step 3: Update **Trebuchet to Lawnchair** *(may need root)*

Step 4: You now have Lawnchair's recents!

***Note: Tried this with debug version of Lawnchair (which you can find in Releases tab on GitHub) too because it has recents options but they are not working since Lawnchair thinks you are still on Trebuchet Launcher's Recents.***

**Comment by HubKing** [comment]
What do I get by doing this? Can I get the "Select" feature of Google Pixel?

https://9to5google.com/wp-content/uploads/sites/4/2022/02/Recents-menu-copy-1.jpg

**Reply by besomuncu** [reply]
@HubKing I think no.

**Reply by HubKing** [reply]
Damn, I need that feature. But do I have to edit the package name? I am using the official LineageOS 21. Changing the package name seems like a hack... and I would have to repeat it every time I update Lawnchair.

**Reply by besomuncu** [reply]
Yeah, unfortunately.

**Comment by Sparxikle** [comment]
> Step 3: Update **Trebuchet to Lawnchair** _(may need root)_

How do you install the app?
I'm keep getting 
Installation failed! Error code: 5
INSTALL_FAILED_UPDATE_INCOMPATIBLE: Existing package com.android.launcher3 signatures do not match newer version; ignoring!

**Reply by faisalbhuiyan3038** [reply]
You need to have root and the Lsposed module Corepatch to install successfully.

**Reply by besomuncu** [reply]
Or a way to delete system apps using ADB

**Comment by HubKing** [comment]
If that's the only way to enable Lawnchair's in-built feature (its own Recents), wouldn't it be more convenient if Lawnchair provides a separate binary with that package name in the release page of this site? I mean, the official version that is distributed on the Play Store probably can't do that, because Google won't allow it, but that should be possible on GitHub, right?

**Comment by curbthepain** [comment]
This is still an issue and now this fix is impossible. You can't rebuild the app anymore. You'd have to fork the project and update the package name yourself.
